src/tutorials/creating-a-pathogen-repo/creating-an-ingest-workflow.rst

```diff
@@ -0,0 +1,465 @@
+
+Additionally, to follow this tutorial, you will need:
+
+* An understanding of `Snakemake <https://snakemake.readthedocs.io/en/stable/>`_ workflows.
+* Pathogen-specific knowledge (e.g. WHO naming scheme) to help with decisions on how to set up the ingest workflow
+
+Setup
+=====
+
+The Nextstrain `pathogen-repo-guide <https://github.com/nextstrain/pathogen-repo-guide>`_ can be used for setting up a
+pathogen repository to hold the files necessary to run and maintain pathogen workflows.
+This tutorial will only focus on using the guide to set up the ingest workflow.
+
+1. Go to the Nextstrain `pathogen-repo-guide repository <https://github.com/nextstrain/pathogen-repo-guide>`_
+2. Follow the `GitHub guide for creating a repository from a template <https://docs.github.com/en/repositories/creating-and-managing-repositories/creating-a-repository-from-a-template>`_.
+3. Follow the `GitHub guide to download the new repository <https://docs.github.com/en/repositories/creating-and-managing-repositories/cloning-a-repository>`_.
+4. Change directory to your new pathogen repository
+
+.. code-block::
+
+    cd <new-pathogen-repository>
+
+Decide on data source
+=====================
+
+The first step for creating an ingest workflow is to decide on the data source for your pathogen's data.
+The pathogen-repo-guide only focuses on downloading public data from `NCBI <https://www.ncbi.nlm.nih.gov/>`_,
+using the rules defined in ``ingest/rules/fetch_from_ncbi.smk``.
+
+.. note::
+
+    If your pathogen does not have sequences on NCBI, then you will need to explore other data sources that are not
+    covered in this tutorial.
+
+NCBI Datasets
+-------------
+
+By default, the pathogen-repo-guide is set to use the `NCBI Datasets CLI <https://www.ncbi.nlm.nih.gov/datasets/docs/v2/reference-docs/command-line/>`_
+tool to download viral sequences using a provided `NCBI taxonomy <https://www.ncbi.nlm.nih.gov/taxonomy>`_ ID.
+This is the simplest route for setting up an ingest workflow, but it is limited to a
+`standard set of fields <https://www.ncbi.nlm.nih.gov/datasets/docs/v2/reference-docs/command-line/dataformat/tsv/dataformat_tsv_virus-genome/#fields>`_
+that is parsed by NCBI Datasets.
+
+You can decide whether NCBI Datasets include sufficient data for your pathogen by inspecting the uncurated data from NCBI Datasets CLI.
+
+.. include:: ../../snippets/uncurated-ncbi-dataset.rst
+
+If the data looks sufficient for your pathogen, then use the NCBI Datasets source in your workflow by
+updating the ``ingest/defaults/config.yaml`` config file.
+
+1. Add your pathogen's NCBI taxonomy ID to the ``ncbi_taxon_id`` parameter
+2. If there are other NCBI Datasets fields you would like to include in the download, you can add them to the ``ncbi_datasets_fields`` parameter
+3. Skip to the :ref:`curation-steps`.
```

Comment:
At least in zika we do some extra steps (`format_ncbi_datasets_ndjson`) - do you want to discuss them here? (Maybe I've missed it)

Reply:
Ah, I had excluded it because it's not a configurable step and seemed like an implementation detail that's not immediately useful to first time users.